docs/queries/sorting.md

```diff
@@ -84,6 +84,8 @@ For example this query will sort by the second tag found in the description.
     sort by tag 2
     ```
 
+Note: this sorting method takes into account any subtags on the global filter.
+
```

Comment:
What does 'takes in to account' mean? Ignore them? Use them? Sort by the sub-tag only, with the parent tag removed?